src/lib/local-api/group.ts

```diff
@@ -0,0 +1,115 @@
+import * as localAPI from '../../types/local-api';
```

Comment:
Why are all of the errors in this file swallowed?

Reply:
At the moment the app isn't using this API but I could remove the catches if you want.

Reply:
I say if it's not used, remove the file. We can add it later if we need it.